addins/Cake.Bridge.yml

```diff
@@ -0,0 +1,11 @@
+Name: Cake.Bridge
```

Comment:
This should be filtered as it is not an addin

Reply:
Done, Cake.Bridge added to blacklist